src/index.ts

```diff
@@ -76,7 +76,7 @@ const storeKey = 'repo';
 
 export const resolveFetch = () => {
     try {
-        if ('fetch' in window) {
+        if (typeof window !== 'undefined' && 'fetch' in window) {
```

Comment:
why not `if(window && 'fetch' in window)`?

Reply:
Checking if `window` is truthy can still trigger TS error "window is not defined"